Proteomics/ProteolyticDigestion/ProteaseDictionary.cs

```diff
@@ -37,18 +38,30 @@ static ProteaseDictionary()
                 {
                     string[] fields = line.Split('\t');
                     List<DigestionMotif> motifList = DigestionMotif.ParseDigestionMotifsFromString(fields[1]);
-
                     string name = fields[0];
                     var cleavageSpecificity = ((CleavageSpecificity)Enum.Parse(typeof(CleavageSpecificity), fields[4], true));
                     string psiMsAccessionNumber = fields[5];
                     string psiMsName = fields[6];
-                    var protease = new Protease(name, cleavageSpecificity, psiMsAccessionNumber, psiMsName, motifList);
-                    dict.Add(protease.Name, protease);
+                    string proteaseModDetails = fields[8];                    
+                    if (proteaseModDetails != "" && proteaseMods != null)
```

Comment:
do we need a check here to see if the dictionary already contains the mod?

Reply:
We never have before, but would be a good check in case someone adds two proteases with the same name to the file. I will make that adjustment.